en/contact-us.md

```diff
@@ -0,0 +1,35 @@
+Title: Contact Us
+table_of_contents: True
+
+# Contact us
+
+We love hearing your feedback and about how you're using MAAS. Get in touch
+with the team from one of the following locations:
+
+- On the [MAAS IRC channel][irc-freenode], hosted on [freenode][freenode].
+- The MAAS [developers mailing list][mailing-list]; a great place to ask
+  questions.
+- Via the [Launchpad][maas-launchpad] source code and issue tracking repository.
```

Comment:
~~Via~~

Reply:
thanks evilnick - I added those because I dislike starting each sentence with 'The'.

Reply:
me too, but it sounds a bit weird